src/Buildalyzer/ProjectAnalyzer.cs

```diff
@@ -209,12 +209,6 @@ private string GetCommand(BuildEnvironment buildEnvironment, string targetFramew
                 // Setting the TargetFramework MSBuild property tells MSBuild which target framework to use for the outer build
                 effectiveGlobalProperties[MsBuildProperties.TargetFramework] = targetFramework;
             }
-            if (Path.GetExtension(ProjectFile.Path).Equals(".fsproj", StringComparison.OrdinalIgnoreCase)
-                && effectiveGlobalProperties.ContainsKey(MsBuildProperties.SkipCompilerExecution))
-            {
-                // We can't skip the compiler for design-time builds in F# (it causes strange errors regarding file copying)
-                effectiveGlobalProperties.Remove(MsBuildProperties.SkipCompilerExecution);

```

Comment:
@dukedagmor Do you remember why you removed this step? Now that I'm running through test failures, I'm seeing the exact behavior described in the comment:

```
Could not copy the file "C:\Code\daveaglick\Buildalyzer\tests\projects\FSharpProject\obj\Debug\netcoreapp3.1\FSharpProject.exe" because it was not found.Process 6788 exited with code 1
```

Before I put this block back in, I'd like to understand the removal so I don't break something else by doing so.

Reply:
i looked on the issues in the fsharp github, there it was mentioned it got solved. Altho it might be broken again since it seems to be a nische issue

Reply:
Ah, got it - so turning this step back on won't necessarily break anything, just might be unnecessary?

Reply:
think so yeah, would need testing tho